src/components/widgets/Map.vue

```diff
@@ -852,6 +860,21 @@ const setHomePosition = async (homePosition: [number, number]): Promise<void> =>
 
 // Allow executing missions
 const executeMissionOnVehicle = async (): Promise<void> => {
+  if (missionStore.showChecklistBeforeArm === true) {
```

Comment:
No need for the `=== true` there.

`if (missionStore.showChecklistBeforeArm) {`